x-pack/plugins/features/server/feature_schema.ts

```diff
@@ -118,8 +119,17 @@ const schema = Joi.object({
   }),
   privilegesTooltip: Joi.string(),
   reserved: Joi.object({
-    privilege: privilegeSchema.required(),
     description: Joi.string().required(),
+    privileges: Joi.array()
```

Comment:
Does this really make sense to have this `description` field outside of the array? It seems like it should be a per item attribute?

Reply:
The `description` field is meant to instruct users how to configure access at the feature level, rather than the privilege level. It's surfaced like this in the UI, so it's not something we can declare on a per-privilege basis:

![image](https://user-images.githubusercontent.com/3493255/78562639-b4a3b600-77e7-11ea-8e47-c21a33e52e29.png)